package/utils/helpers.js

```diff
@@ -39,6 +39,9 @@ const canProcess = (rule, fn) => {
   return null
 }
 
+const runningWebpackDevServer = process.env.WEBPACK_DEV_SERVER &&
+  process.env.WEBPACK_DEV_SERVER !== 'undefined'
```

Comment:
Is `undefined` a string? And not the `undefined` keywork? 🤔

```node
> process.env.WEBPACK_DEV_SERVER
undefined
> process.env.WEBPACK_DEV_SERVER !== 'undefined'
true
> process.env.WEBPACK_DEV_SERVER !== undefined
false
>
```

```suggestion
// When running the webpack-dev-server, this ENV value is set.
const runningWebpackDevServer = process.env.WEBPACK_DEV_SERVER !== undefined
```

Reply:
Updated to check against 'true'.